template/deploy/helm/[[operator]]/templates/deployment.yaml.j2

```diff
@@ -48,6 +48,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.annotations['internal.stackable.tech/image']
+            - name: CLUSTER_DNS_DOMAIN
```

Comment:
Any comments on how you got this name? I personally find `KUBERNETES_SERVICE_DNS_DOMAIN` more speaking (as it's the *service* domain, not the cluster domain I guess)

Reply:
No strong opinion, will change. Just did not want to use Kubernetes (due to openshift, although they have the env vars alla KUBERNETES_SERVICE_HOST as well... will change

Reply:
https://github.com/stackabletech/operator-templating/pull/445/commits/ea6761bbbd88b1729fb574bdf40dbf47b2c83348

Reply:
Many thanks!